components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Protocol/src/Services/SubscriptionServices.cs

```diff
@@ -1022,24 +1022,25 @@ private class MonitoredItemWrapper {
             /// <summary>
             /// Create new stack monitored item
             /// </summary>
-            /// <param name="session"></param>
+            /// <param name="messageContext"></param>
+            /// <param name="typeTree"></param>
             /// <param name="codec"></param>
             /// <param name="activate"></param>
             /// <returns></returns>
-            internal void Create(Session session, IVariantEncoder codec, bool activate) {
+            public void Create(ServiceMessageContext messageContext, ITypeTable typeTree, IVariantEncoder codec, bool activate) {
```

Comment:
In the simplified event configuration I need access to the NodeCache, so use NodeCache as the parameter instead

Reply:
Good catch! I will fix.

Reply:
Now resolved.